packages/conf-ncurses/conf-ncurses.1/opam

```diff
@@ -7,6 +7,9 @@ license: "MIT"
 build: [
   ["pkg-config" "ncurses"] { os != "darwin"}
 ]
+depends: [
+  "conf-pkg-config" { os != "darwin"}
```

Comment:
You're right the filter is not allowed. I quickly fixed this by removing the filter, but @dsheets proposition might be a better solution.